renderer/style.css

```diff
@@ -13,23 +13,27 @@ body,
 
 /* File picker (Main screen) */
 .file-picker {
+  width: 50%;
+  height: 50%;
```

Comment:
Wait, why is this changed to 50%? The "Choose files" text is no longer in the center now.